guides/common/modules/proc_installing-and-configuring-puppet-agent-manually.adoc

```diff
@@ -12,16 +12,13 @@ endif::[]
 .Prerequisites
 * The host must have a Puppet environment assigned to it.
 ifdef::satellite[]
-* The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
-The `puppet-agent` rpm is available within the following repositories:
-** {Team} {project-client-name} for RHEL 7 Server RPMs x86_64 {endash} `{RepoRHEL7ServerSatelliteToolsProductVersion}`
-** {Team} {project-client-name} for RHEL 8 Server RPMs <arch> {endash} `{RepoRHEL8ServerSatelliteToolsProductVersion}`
-** {Team} {project-client-name} for RHEL 9 Server RPMs <arch> {endash} `{RepoRHEL9ServerSatelliteToolsProductVersion}`
+* The *{project-client-name}* repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
+The `puppet-agent` RPM is available within the following repositories:
+** {Team} {project-client-name} (for RHEL 7 Server) (RPMs) x86_64 {endash} `{RepoRHEL7ServerSatelliteToolsProductVersion}`
+** {Team} {project-client-name} for RHEL 8 x86_64 (RPMs) {endash} `{RepoRHEL8ServerSatelliteToolsProductVersion}`
+** {Team} {project-client-name} for RHEL 9 x86_64 (RPMs) {endash} `{RepoRHEL9ServerSatelliteToolsProductVersion}`
```

Comment:
Same here: AFAIk we have multiple arches and we should list them instead of implying we only have x86_64.

Reply:
We only provide puppet-agent for x86_64 if you take a look into ohsnap.

Reply:
We can ignore i686 for RHEL 6, but I also see `aarch64` for RHEL 8.

Reply:
I'll remove this from here altogether and fix [BZ2152621](https://bugzilla.redhat.com/show_bug.cgi?id=2152621) instead. I'll unify the prerequisites with other modules.